rust/Cargo.toml.in

```diff
@@ -67,6 +67,8 @@ time = "=0.3.20"
 
 suricata-derive = { path = "./derive", version = "@PACKAGE_VERSION@" }
 
+suricata-lua-sys = { git = "https://github.com/jasonish/suricata-lua-sys", version = "0.1.0-alpha.1" }
```

Comment:
This repo contains a copy of the Lua source. We could also use a git sub-module, but that does open us up to breakage by upstream still. So I opt to create a copy, so we're in full control.